modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -1332,6 +1335,19 @@ private boolean hasBearerMethods(List<CodegenSecurity> authMethods) {
         return false;
     }
 
+    // hasHttpSignatureMethods returns true if the specified OAS model has
+    // HTTP signature methods.
+    // The HTTP signature scheme is defined in https://datatracker.ietf.org/doc/draft-cavage-http-signatures/
+    private boolean hasHttpSignatureMethods(List<CodegenSecurity> authMethods) {
```

Comment:
Hey what if you make this `protected boolean`?
Then subclasses can use it.

Reply:
Or how about putting it here?
modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ProcessUtils.java
That way both classes can use it.

Reply:
I moved it to ProcessUtils.java